src/RemoveTrivialForLoops.cpp

```diff
@@ -10,12 +10,18 @@ namespace Internal {
 class RemoveTrivialForLoops : public IRMutator {
     using IRMutator::visit;
 
+    bool device_loops;
```

Comment:
It wouldn't... in this branch :)

It gets used to remove trivial loops, even if they are device loops, in the hexagon branch. If you don't want to merge this because it isn't used, I'm not opposed to just leaving it for later.